ASTProcessor/src/test/java/edu/usb/argos/ASTProcessor/AttributeAnalyzerTest/JavaAttributeVisitorTest.java

```diff
@@ -0,0 +1,238 @@
+package edu.usb.argos.ASTProcessor.AttributeAnalyzerTest;
+
+import edu.usb.argos.ASTProcessor.visitor.infraestructure.antlr.visitors.method.AttributeHandler;
+import edu.usb.argos.ASTProcessor.visitor.core.entities.method.AttributeInfo;
+import edu.usb.argos.ASTProcessor.visitor.infraestructure.antlr.visitors.method.JavaAttributeVisitor;
+import org.antlr.v4.runtime.CharStream;
+import org.antlr.v4.runtime.CharStreams;
+import org.antlr.v4.runtime.CommonTokenStream;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import edu.usb.argos.ASTProcessor.antlr.JavaLexer;
+import edu.usb.argos.ASTProcessor.antlr.JavaParser;
+import java.util.List;
+import java.util.Optional;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
It's not good to use * when importing due to we access to the all library so please import just what you need.

Reply:
Done